packages/hint/docs/user-guide/troubleshoot/summary.md

```diff
@@ -1,107 +1,109 @@
-# Common issues when installing or running webhint
+---
+date: 08/26/2020
+---
+# Troubleshoot webhint installation and runtime issues
 
-The team currently supports webhint on the latest 2 LTS and
-current branch. While it is possible to run it on x86 version of
-Node.js, it is recommended to use the x64 one. The reason is that
-the project still uses some binary packages (`canvas` and `iltorb`)
-and in most cases there are precompiled versions for x64 and thus
-avoiding to compile anything.
+This topic contains the most common issues reported by users along with potential fixes.  If you encounter an undocumented error, please [open an issue][GitHubWebhintioHintNew] on GitHub.
 
-This section contains the most commons issues reported by users with
-potential fixes. If you run into something that is not documented
-here please [open an issue in the hint repo][new issue].
+> [!NOTE]
+> `webhint` is supported on the most recent LTS and Current version of Node.js.  You should use the x64 version of webhint.
 
 ## Windows Subsystem Linux
```

Comment:
@molant @antross 
Section is out of date, please provide updated instructions for WSL 2.

Reply:
@molant @antross
Is there an update?

Reply:
Will open a separate PR to track WSL2 docs updates. Thanks!